modules/installation-aws-permissions-iam-roles.adoc

```diff
@@ -53,10 +60,24 @@ The control plane and compute machines require the following IAM role permission
 * `kms:DescribeKey`
 ====
 
-.Required IAM role permissions for compute instance profiles
+.Default IAM role permissions for worker instance profiles
 [%collapsible]
 ====
-* `sts:AssumeRole`
 * `ec2:DescribeInstances`
 * `ec2:DescribeRegions`
 ====
+
+= Custom permissions for IAM Instance Profiles
+
+Depending on your cluster set of operators and their configuration, it may be necessary to grant additional permissions to the instance profiles. Instead of manually adding each permission, you may https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_generate-policy.html?icmpid=docs_iam_console[generate policies based on access activity].
```

Comment:
> Depending on your cluster set of operators and their configuration, it may be necessary to grant additional permissions to the instance profiles.

@faermanj can we review that section/sentence? We expect that the permissions provided to the operators are set by credentials provided by CCO using mint or manual (using IAM User or IAM Roles) - generally defined on `CredentialsRequests` for Red Hat Operators.

This is one ongoing PR to keep it more clear: https://github.com/openshift/openshift-docs/pull/51541

The other reason that the permissions set to the services running on OCP should not affect the instance profile is that the SDN blocks access to the metadata service, to avoid sharing EC2 permissions from the Instance Profile being used by the pods.

Reply:
cc @jeana-redhat

Reply:
@mtulio Thanks, i'll check a better wording. What i meant is that, if you add an operator that uses other services, you'd need to have those permissions in the roles...